Frontend/android-native/app/src/main/java/in/juspay/mobility/CommonJsInterface.java

```diff
@@ -3627,11 +3634,71 @@ private LatLng getNewLatLng(float fraction, LatLng a, LatLng b) {
         return new LatLng(lat, lng);
     }
 
+
+    private  void animatePolyline(final int staticColor, JSONObject androidPolylineAnimationConfigObject){
+        try{
+            int animateColor = Color.parseColor(androidPolylineAnimationConfigObject.getString("color"));
+            int drawingDuration = androidPolylineAnimationConfigObject.getInt("drawingDuration");
+            int fadingDuration = androidPolylineAnimationConfigObject.getInt("fadingDuration");
+            int delayDuration = androidPolylineAnimationConfigObject.getInt("delayDuration");
+
+            polylineAnimatorSet =  new AnimatorSet();
+
+            polylineDrawingAnimator = ValueAnimator.ofInt(0, 100);
```

Comment:
Can we declare the variables - polylineDrawingAnimator, polylineColorFadingAnimator locally?